lib/managers/folders.js

```diff
@@ -321,6 +321,40 @@ Folders.prototype.getTrashedFolder = function(folderID, qs, callback) {
 	this.client.get(apiPath, params, this.client.defaultResponseHandler(callback));
 };
 
+/* Restores an item that has been moved to the trash. Default behavior is to restore the item
+ * to the folder it was in before it was moved to the trash. If that parent folder no longer
+ * exists or if there is now an item with the same name in that parent folder, the new parent
+ * older and/or new name will need to be included in the request.
+ *
+ * API Endpoint: '/folders/:folderID'
+ * Method: POST
+ *
+ * @param {string} folderID - The ID of the folder to restore
+ * @param {?string} [name] - The new name for this item
+ * @param {string} [parentFolderID] - The new parent folder for this item
+ * @param {Function} callback - Called with folder information if successful, error otherwise
+ * @returns {void}
+ */
+Folders.prototype.restoreFolder = function(folderID, name, parentFolderID, callback) {
```

Comment:
I think `restoreFolder` is redundant, since this is already in the Folders manager.  We should rename to `restoreFromTrash` for clarity.